src/Components/AddPlace.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import '../AddPlace.css';
+
+function AddPlace() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.